src/Client/Miscellaneous.php

```diff
@@ -47,7 +46,7 @@ public function getLanguageCodes(): LanguageCodeList
     public function getInvoiceCheckout(
         string $invoiceId,
         ?string $lang
-    ): InvoiceCheckoutHTML {
+    ): string {
```

Comment:
If we still want to return a result class like with the other results you could keep `InvoiceCheckoutHTML` but make it not extend Abstract result but yeah not needed for any functionality, although we could in the future make something like getIframe() which returns it wrapped in <iframe> but then again, maybe that is not enough to justify a class :D

What's thinking on this, why did you go for string and not keep the class?

Reply:
Embarassed to admit I didn't think of not extending the class 🤦

Will put it back in lol

Reply:
Done!